site/app/models/gradeable/LateDayInfo.php

```diff
@@ -282,6 +289,14 @@ public function getLateDayException() {
         return $this->late_day_exceptions;
     }
 
+    /**
+     * Gets the reason for an excused absence extension
+     * @return string
+     */
+    public function getReasonForException(): string {
```

Comment:
No need to specify the return type in a PHPDoc here because it doesn't provide any additional information.
```suggestion
    /**
     * Gets the reason for an excused absence extension
     */
    public function getReasonForException(): string {
```